Blockzilla.xcodeproj/xcshareddata/xcschemes/Focus.xcscheme

```diff
@@ -55,6 +56,7 @@
       buildConfiguration = "FocusDebug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = ""
```

Comment:
is this supposed to be in here?

Reply:
That is a great question. The migrator added it for me. Not sure what it controls

Reply:
ok because if it wasn't there before.... most likely it's not needed now? is there a way to take out?

Reply:
im not sure if it's like a new thing or an extraneous thing

Reply:
yeah I agree. I'm trying to look up what this property is used for now